klog.go

```diff
@@ -424,10 +426,12 @@ func InitFlags(flagset *flag.FlagSet) {
 	}
 
 	flagset.StringVar(&logging.logDir, "log_dir", logging.logDir, "If non-empty, write log files in this directory")
-	flagset.StringVar(&logging.logFile, "log_file", logging.logFile, "If non-empty, use this log file")
+	flagset.StringVar(&logging.logFile, "log_file", logging.logFile, "If non-empty, use this log file (mutually exclusive with the log-dir option)")
 	flagset.Uint64Var(&logging.logFileMaxSizeMB, "log_file_max_size", logging.logFileMaxSizeMB,
 		"Defines the maximum size a log file can grow to. Unit is megabytes. "+
 			"If the value is 0, the maximum file size is unlimited.")
+	flagset.IntVar(&logging.logFileMaxBackups, "log_file_max_backups", logging.logFileMaxBackups,
```

Comment:
`"log_file_max_backups"` should be `"klog_log_file_max_backups"` here @sayanibhattacharjee

Reply:
Fixed this.